<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedMethodsWidgetLayout</class>
 <widget class="QWidget" name="AdvancedMethodsWidgetLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>431</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>32767</width>
    <height>32767</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advanced methods</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>2</number>
       </property>
       <item>
        <widget class="QLabel" name="method_type_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>27</height>
          </size>
         </property>
         <property name="text">
          <string>Method:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="method_combo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>270</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32767</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="grid_gbox">
       <property name="title">
        <string>Grid</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="hor_spacing_label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Horizontal spacing (μm):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="hor_spacing_ledit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="move_hor_label">
          <property name="text">
           <string>Move horizontal:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="move_left_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Left2.png</normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Left2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QPushButton" name="move_right_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Right2.png</normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Right2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="ver_spacing_label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>Vetical spacing (μm):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="ver_spacing_ledit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>27</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QLabel" name="move_ver_label">
          <property name="text">
           <string>Move vertical:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="move_up_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Up2.png</normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Up2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <widget class="QPushButton" name="move_down_button">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>27</width>
            <height>27</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Down2.png</normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Down2.png</iconset>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="adjust_size_cbox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>27</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>27</height>
             </size>
            </property>
            <property name="text">
             <string>Adjust size</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="fast_axis_label">
              <property name="text">
               <string>fast axis:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="vertical_radio_button">
              <property name="text">
               <string>vertical</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="horizontal_radio_button">
              <property name="text">
               <string>horizontal</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="3" column="0" colspan="6">
         <widget class="QTreeWidget" name="grid_treewidget">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
           </font>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Lines</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Frames per line</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="4" column="0" colspan="3">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="nimages_per_point_label">
              <property name="text">
               <string>Images per point:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="nimages_per_point_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Number of images per grid point</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="npasses_label">
              <property name="text">
               <string>Number of passes:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="npasses_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Number of passes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="dark_time_between_passes_label">
              <property name="text">
               <string>Dark time between passes:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="dark_time_between_passes_ledit">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>27</height>
               </size>
              </property>
              <property name="toolTip">
               <string>dark time between passes</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="5" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="overlay_cbox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Display overlay:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="overlay_slider">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximum">
             <number>255</number>
            </property>
            <property name="value">
             <number>120</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="overlay_color_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Overlay color</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0" colspan="4">
         <layout class="QHBoxLayout" name="buttons_hlayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="draw_grid_button">
            <property name="text">
             <string>Draw</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Grid.png</normaloff>../../../../../../../../home/experiences/proxima2a/Icons/Grid.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="move_to_grid_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Move to center</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_grid_button">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>overlay_cbox</sender>
   <signal>toggled(bool)</signal>
   <receiver>overlay_slider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>vertical_radio_button</sender>
   <signal>toggled(bool)</signal>
   <receiver>horizontal_radio_button</receiver>
   <slot>raise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>401</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
